WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteViewModel.kt

```diff
@@ -349,15 +349,31 @@ class MySiteViewModel @Inject constructor(
         }
     }
 
-    private fun getPositionOfQuickStartItem(siteItems: Map<MySiteTabType, List<MySiteCardAndItem>>) =
+    private fun getPositionOfQuickStartItem(
+        siteItems: Map<MySiteTabType, List<MySiteCardAndItem>>,
+        activeTask: QuickStartTask
+    ) =
             if (isMySiteTabsEnabled) {
-                (siteItems[MySiteTabType.SITE_MENU] as List<MySiteCardAndItem>)
-                        .indexOfFirst { it.activeQuickStartItem }
+                val currentTab = orderedTabTypes[_selectTab.value!!.peekContent().position]
```

Comment:
👋  @AjeshRPai - It's preferable not to use double bang if there are other approaches to managing nullability. Can this be worked to something like the following?

```
_selectTab.value?.let {
  ...
}
```

Reply:
Yup right. Missed this. I have made the changes in the [09e34c3](https://github.com/wordpress-mobile/WordPress-Android/pull/16267/commits/09e34c3972e4279cb6dbb50cfe94dda28bee0a1e). Please have a look.